src/EnergyPlus/UnitVentilator.cc

```diff
@@ -2058,7 +2054,7 @@ namespace UnitVentilator {
                     if (state.dataUnitVentilators->UnitVent(UnitVentNum).CoilOption != state.dataUnitVentilators->NoneOption) {
                         if (state.dataSize->ZoneHVACSizing(zoneHVACIndex).CoolingSAFMethod > 0) {
                             SAFMethod = state.dataSize->ZoneHVACSizing(zoneHVACIndex).CoolingSAFMethod;
-                            SizingMethod = CoolingAirflowSizing;
+                            SizingMethod = static_cast<int>(AutoSizingType::CoolingAirFlowSizing);
                             ZoneEqSizing(state.dataSize->CurZoneEqNum).SizingMethod(SizingMethod) = SAFMethod;
                             if (SAFMethod == None || SAFMethod == SupplyAirFlowRate || SAFMethod == FlowPerFloorArea ||
```

Comment:
These conditionals where you are comparing a single variable to multiple enum values should be either `switch/case` or `BITF`.

Reply:
Absolutely.